### ISSUE TITLE
VAMC Medical Records: adopt V1 and V3 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16960]
## Description
VAMC Medical Records pages need the following updates to components:

Example pages:

- [Boston Medical Records](https://www.va.gov/boston-health-care/medical-records-office/)
- [Minneapolis Medical Records](https://www.va.gov/minneapolis-health-care/medical-records-office/)

### Elements that are not using components that should be
1. ~Telephone~
   - ~Location Phone numbers are using the component but we want to make sure exts are being parsed correctly~
2. ~Link~
   - ~Facility names should use the link component~
3. [Back to top component](https://design.va.gov/components/back-to-top)
   - Update 5/3: After determining that this component does not actually have built in intelligence to only show based on page height:
      -  This should still be added, the Top task pages are on the longer side so having this on these pages will probably make sense more often than not. 

### Not in scope: V1 components that need to be upgraded - DONE
1. ~Alert~
   - ~Currently using the v1 component but needs to be updated [to v3](https://design.va.gov/components/alert#examples---standard---v3)~
2. ~Accordion~
   - ~Medical records accordion section is using v1 component and needs to be updated [to v3](https://design.va.gov/components/accordion#examples---v3).~
3. ~Button~
   - ~It is inside the alert and is currently using the v1 component but needs to be updated [to v3](https://design.va.gov/components/button/#examples---v3).~

## Acceptance Criteria
- [ ] Back to top component has been
- [ ] Update and run existing test(s).
- [ ] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

**Comment by maxx1128** [comment]
I estimate this ticket at a **5** for moving over a good amount of relatively simple elements over to components but also upgrading a few extra ones. It's another "If I could make it a 4, I would" scenario, so I'm rounding up.

**Comment by laflannery** [comment]
Alert and button are already v3:
![Screenshot 2024-03-27 at 10 53 58 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/1a03958d-26a6-4201-b3e2-2ca2b26aa2fe)

My thought is that it happened with this [PW ticket for the CTA widget](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17307)

**Comment by laflannery** [comment]
Accordions have also already been updated to V3:
![Screenshot 2024-03-27 at 10 59 25 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/e91d3e83-c056-4636-80e6-21357ea386bb)

I don't know when that would have been done but I can probably track it down

**Comment by laflannery** [comment]
Last update: I removed the link component from this ticket, after working with PW, the links I noted are CMS content and we are limited with what we can do with that type of content

**Comment by eselkin** [comment]
Links in text are in the form of wysiwyg paragraphs so drupal content. The non-clinical service location paragraphs are handled by the Records ticket. It's likely only the back-to-top will be left, but will say 2 just in case.

**Comment by laflannery** [comment]
Confirmed on prod, closing
